package HomeWorkExceptions.Task_2;

public class Run {

    public static void main(String[] args) throws Exception {

        CheckUncheckNumber chek = new CheckUncheckNumber();
        chek.Check(100);
    }
}
